js/main.js

```diff
@@ -36,15 +36,15 @@ function generate_label_html (labels) {
 }
 
 function create_issue_url (issue) {
-  var loc = window.location.href;
+  const loc = window.location.href;
   // if the location ends with a slash
   if (loc.slice(-1) == '/') {
     return loc + 'issue.html?issue=' + issue;
   } else if (loc.indexOf('?') > 0){
     // if a url query has been applied, match the last / and replace everything onwards with the issue url
     return loc.replace(/(?:\/[^\/\r\n]*)$/, '/') + 'issue.html?issue=' + issue;
   }
-  return loc.replace(/\w+\.[^\.]+$/, '') + 'issue.html?issue=' + issue;
```

Comment:
Do you not need to escape the `.` when you're in a "not-group" (in lack of better terms -- i.e. `[^...]`)?

Reply:
yep. it's totally not required - inside character class, every characters except `^`, `-`, `]`,`\` will be interpreted as literal(good article pointing to this - https://www.regular-expressions.info/charclass.html - first paragraph of "Metacharacters Inside Character Classes" section). 

So `\.` and `.` don't have any difference here. I've removed this backslash just because of jslint warning saying it's redundant and can be removed. )

Reply:
I see! Thanks for the explanation!